acceptance/tests/language/invoke_pp_function_from_independent_module.rb

```diff
@@ -0,0 +1,50 @@
+test_name 'Puppet-language functions can be called from outside of any module' do
+
+  mini_func = 'function jenny::mini($a, $b) {if $a <= $b {$a} else {$b}}'
+  maxi_func = 'function jenny::nested::maxi($a, $b) {if $a >= $b {$a} else {$b}}'
+  
+  step 'Create PP functions on master' do
+    manifest = <<-EOM.gsub(/^ {6}/,'')
+      file {['/etc/puppetlabs',
+          '/etc/puppetlabs/code',
+          '/etc/puppetlabs/code/modules',
+          '/etc/puppetlabs/code/modules/jenny',
+          '/etc/puppetlabs/code/modules/jenny/functions',
+          '/etc/puppetlabs/code/modules/jenny/functions/nested']:
+        ensure => directory,
+        owner => 'puppet',
+        group => 'puppet',
+        mode => '0755',
+      }
+      
+      file {'/etc/puppetlabs/code/modules/jenny/functions/mini.pp':
```

Comment:
IMO, this test file should be merged with the one above.    acceptance runtimes are an issue and this one repeats a bunch of the setup steps as well as the test for mini function.